README.md

```diff
@@ -975,18 +974,31 @@ def test_backup_action():
 
     # If you didn't declare do_backup in the charm's metadata, 
     # the `ConsistencyChecker` will slap you on the wrist and refuse to proceed.
-    out: scenario.ActionOutput = ctx.run_action(ctx.on.action("do_backup"), scenario.State())
+    state = ctx.run(ctx.on.action("do_backup"), scenario.State())
 
-    # You can assert action results, logs, failure using the ActionOutput interface:
-    assert out.logs == ['baz', 'qux']
-    
-    if out.success:
-      # If the action did not fail, we can read the results:
-      assert out.results == {'foo': 'bar'}
+    # You can assert action results and logs using the action history:
+    assert ctx.action_history[0].logs == ['baz', 'qux']
```

Comment:
will the user ever want to access anything other than index 0 in this history?

In principle we want Context to be only used once to fire an event, so there should never be more than 1 item in action_history.
So is it really a history? 
Or are we better off calling it `Context.task`, pop a warning (or an exception even?) if someone overrides an existing task by running two action events on the same Context?

This would simplify the internal usage as well as the user-facing API.

WDYT?

Reply:
I felt like there could be a use case here, but both you and @benhoyt pushed back on this, and I think I was trying too hard to predict future uses, so I'm in agreement to simplify this.